Gemfile

```diff
@@ -5,6 +5,8 @@ ruby '3.2.2'
 
```

Comment:
- [ **OPTIONAL** ] `README` file: To make your README be more pprofessinal and descriptive, I think it would be better if you also update the `Run Tests` content information with a correct command of how to run your specs tests i.e. to add the following command:
```
### Run tests
To run tests, run the following command:
 rspec spec/
```